sw/device/tests/alert_handler_escalation_test.c

```diff
@@ -114,24 +91,79 @@ bool test_main(void) {
 
   CHECK_DIF_OK(dif_keymgr_init(
       mmio_region_from_addr(TOP_EARLGREY_KEYMGR_BASE_ADDR), &keymgr));
+}
+
+/**
+ * Configure the alert handler. The escalation phases (NMI interrupt, LC scrap
+ * state, chip reset) are assigned to alert class A.
+ */
+static void config_alert_handler(void) {
+  // Escalation phase 0 is the NMI interrupt whose timeout value before
+  // progressing to phase 1 differs depending on the simulation device. For
+  // example, on the ChipWhisperer 1000000 cycles are enough to prevent a
+  // premature cancellation of the NMI interrupt handler (see
+  // `ottf_external_nmi_handler`).
+  uint32_t phase0DurationCycles = 1000000;
+  if (kDeviceType == kDeviceSimVerilator) {
+    phase0DurationCycles /= 10;
+  } else if (kDeviceType == kDeviceSimDV) {
+    phase0DurationCycles /= 100;
+  }
+
+  dif_alert_handler_escalation_phase_t escalationProfiles[] = {
+      {.phase = kDifAlertHandlerClassStatePhase0,
+       .signal = 0,
```

Comment:
@pamaury for multitop we might need to make these signal codes constants in the top since they're used semantically (i.e. we write 0 to mean NMI, 1 to mean escalation LC state, 3 to mean chip reset). Tops could change the meanings of these numbers.

Reply:
Thanks for the reminder @jwnrt . I am aware of this problem and we discussed it with @matutem in the past. I don't remember the answer but I think it wasn't clear how/if this is configured in the Hjson or just how we turn out to implement it. In any case this is definitely something we want for multitop if we can abstract it nicely.